lib/dry/types/hash/schema.rb

```diff
@@ -149,8 +149,6 @@ def resolve(hash)
           end
         end
       end
-
-      private_constant(*constants(false))
```

Comment:
@backus k, this wasn't hard https://github.com/dry-rb/dry-types/pull/147/files#diff-6f163e52d96701446ae4747de65b62ebR51 :)
I restored the initial version `private_constant(*constants(false)) `